benchmarks/benchmark/tools/dlio/README.md

```diff
@@ -16,12 +16,19 @@ If you need to reinstall any resources, make sure to delete this file as well.
 
 ## Run DLIO Job
 1. Update the `variables.tf` file with your desired settings to run your machine learning benchmark workload
-2. Change the dlio image in `dlio/podspec.tpl`
+2. Change the dlio image in `modules/dlio/podspec.tpl`
 3. Run `terraform init`
 4. Run `terraform apply`
 5. After you finish your test, run `terraform destroy` to delete the
    resources
 
+## Run DLIO Job with Parallelstore
+1. update `variables.tf` file with your desired settings to run your machine learning benchmark workload, notably set `gcs_fuse_csi_driver_enabled` to `false` and `paralllestore_csi_driver_enabled` to `true`. If you want to use static provisioning, update the "parallelstore variables" and `parallelstore_storageclass` to `""`.
```

Comment:
also mention which dlio image has this terraform been tested with

Reply:
done.